test/remote/gateways/remote_priority_test.rb

```diff
@@ -2,323 +2,268 @@
-
-    # purchase params fail missing card verification number
-    @credit_card_purchase_fail_missing_verification = credit_card('4111111111111111', month: '01', year: '2029', first_name: 'Marcus', last_name: 'Rashford', verification_value: '')
+  def test_successful_authorize
+    response = @gateway.authorize(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal 'Approved or completed successfully', response.message
+  end
 
-    # authorize params success
-    @amount_authorize = 799
-    # authorize params success end
+  def test_failed_authorize
+    response = @gateway.authorize(@amount, @invalid_credit_card, @options)
+    assert_failure response
+    assert_equal 'Invalid card number', response.message
   end
 
   def test_successful_purchase
-    response = @gateway.purchase(@amount_purchase, @credit_card, @option_spr)
+    response = @gateway.purchase(@amount, @credit_card, @options)
     assert_success response
-    assert_equal 'Approved', response.params['status']
+    assert_equal 'Approved or completed successfully', response.message
   end
 
-  # Invalid card number
   def test_failed_purchase
-    response = @gateway.purchase(@amount_purchase, @credit_card_purchase_fail_invalid_number, @option_spr)
+    response = @gateway.purchase(@amount, @invalid_credit_card, @options)
     assert_failure response
-
     assert_equal 'Invalid card number', response.message
-    assert_equal 'Declined', response.params['status']
   end
 
-  # Missing card number month
   def test_failed_purchase_missing_card_month
-    response = @gateway.purchase(@amount_purchase, @credit_card_purchase_fail_missing_month, @option_spr)
-    assert_failure response
+    card_without_month = credit_card('4242424242424242', month: '')
+    response = @gateway.purchase(@amount, card_without_month, @options)
 
+    assert_failure response
     assert_equal 'ValidationError', response.error_code
-    assert_equal 'Validation error happened', response.params['message']
     assert_equal 'Missing expiration month and / or year', response.message
   end
 
-  # Missing card verification number
   def test_failed_purchase_missing_card_verification_number
-    response = @gateway.purchase(@amount_purchase, @credit_card_purchase_fail_missing_verification, @option_spr)
-    assert_failure response
+    card_without_cvv = credit_card('4242424242424242', verification_value: '')
+    response = @gateway.purchase(@amount, card_without_cvv, @options)
 
+    assert_failure response
     assert_equal 'CVV is required based on merchant fraud settings', response.message
-    assert_equal 'Declined', response.params['status']
   end
 
-  # Authorize tests
-  def test_successful_authorize
-    response = @gateway.authorize(@amount_purchase, @credit_card, @option_spr)
-    assert_success response
-    assert_equal 'Approved', response.params['status']
+  def test_successful_capture
```

Comment:
This is a minor nitpick, but I think it's conventional to name tests that call multiple methods after both methods, e.g. `test_successful_authorize_and_capture`

Reply:
Thank you for this minor but helpful nitpick 😄 . I will change it.